plugins/importexport/native/filter/NativeXmlArticleFilter.inc.php

```diff
@@ -89,6 +89,12 @@ function populateObject($submission, $node) {
 				$submission->setSectionId($section->getId());
 			}
 		}
+		$datePublished = $node->getAttribute('date_published');
+		$issue = $deployment->getIssue();
```

Comment:
An import could also be just import of one published article that would then have the child element `issue_identification`, that would be handled later i.e. there would be no `$issue` at this place yet. Thus, could we maybe also check if the child element exists and consider it in the `if` statement below i.e. if there is child element `issue_identification` and no `$datePublished` add the error?